src/collectd.conf.pod

```diff
@@ -8300,6 +8300,11 @@ Synopsis:
      CollectVCL         false
      CollectVSM         false
      CollectWorkers     false
+     CollectLCK         false
```

Comment:
The length of config option names is not constrained. If "LCK" is an acronym, it can stay as it is. If it is short for "Lock", please expand the name for readability.